ena_portal_api/ena_handler.py

```diff
@@ -234,15 +231,13 @@ def get_study(
         data["fields"] = fields or STUDY_DEFAULT_FIELDS
 
         if primary_accession and not secondary_accession:
-            data["query"] = 'study_accession="{}"'.format(primary_accession)
```

Comment:
I think we could also simplify this as such:

```
def get_study(self, primary_accession=None, secondary_accession=None, fields=None, attempt=0):
    """Retrieve study information based on primary or secondary accession."""
    data = {
        **get_default_params(),
        "result": "read_study",
        "fields": fields or STUDY_DEFAULT_FIELDS,
    }

    query_parts = []
    if primary_accession:
        query_parts.append(f'study_accession="{primary_accession}"')
    if secondary_accession:
        query_parts.append(f'secondary_study_accession="{secondary_accession}"')

    data["query"] = " AND ".join(query_parts)

 
```

Reply:
Yep